docs/images/logo-title.svg

```diff
@@ -0,0 +1,17 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="762" width="548">
```

Comment:
I think you should update the README link as well: https://github.com/kubeflow/katib/blame/master/README.md#L2.

Reply:
Updated that link, and also added pngs of both SVG files as `logo.png` and `logo-title.png`, to match the SVG filenames.

For future reference, making the pngs was pretty easy, if they ever get edited again:

    cd docs/images/
    inkscape logo.svg -o logo.png
    inkscape logo-title.svg -o logo-title.png